chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -5,4 +5,21 @@
 
 #include "brave/browser/brave_profile_prefs.h"
 #include "brave/browser/brave_local_state_prefs.h"
-#include "../../../../chrome/browser/prefs/browser_prefs.cc"
+#include "third_party/widevine/cdm/buildflags.h"
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+#include "brave/browser/widevine/widevine_utils.h"
+#endif
+
+#define MigrateObsoleteProfilePrefs MigrateObsoleteProfilePrefs_ChromiumImpl
+#include "../../../../chrome/browser/prefs/browser_prefs.cc"  // NOLINT
+#undef MigrateObsoleteProfilePrefs
+
+void MigrateObsoleteProfilePrefs(Profile* profile) {
+  MigrateObsoleteProfilePrefs_ChromiumImpl(profile);
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+  MigrateWidevinePrefs(profile);
```

Comment:
In the original Chromium function they have a nice tradition to write a comment about when this was added, I'd suggest we do the same :) And also mention that `// This method should be periodically pruned of year+ old migrations.` above

Reply:
Done.